tests/unit/test_configuration.py

```diff
@@ -327,3 +328,27 @@ def test_validate_miswrite_zipcode_digit_probabilities_failures(probabilities, m
                 },
             },
         )
+
+
+def test_get_config(caplog):
+    user_config = {
+        DATASETS.acs.name: {
+            Keys.COLUMN_NOISE: {
+                "zipcode": {
+                    NOISE_TYPES.missing_data.name: {
+                        Keys.PROBABILITY: 0.25,
+                    },
+                },
+            },
+        },
+    }
+    config_1 = get_config()
+    assert isinstance(config_1, dict)
+    assert not caplog.records
+
+    config_2 = get_config("decennial_census", user_config)
+    assert isinstance(config_2, dict)
+    assert len(caplog.records) == 1
```

Comment:
Oh nice, so this did work! 

I'd recommend tightening up the test by checking that the warning that got thrown is indeed the correct one. I don't recall exactly what it would looks like, but something like

```
assert "not a valid option" in caplog.records`
```